clientv3/watch.go

```diff
@@ -18,6 +18,7 @@ import (
 	"context"
 	"errors"
 	"fmt"
+	"go.uber.org/zap"
```

Comment:
move this to line 28.

Reply:
@jingyih 
Your comment has been addressed.